kitsune/wiki/tests/test_templates.py

```diff
@@ -206,6 +206,13 @@ def test_document_fallback_no_translation(self):
         doc('#doc-pending-fallback').remove()
         # Included content is English.
         eq_(pq(r.document.html)('div').text(), doc('#doc-content div').text())
+        
+        r.is_ready_for_localization=True
+        
+        # Fallback message is not shown.
+        eq_(None, len(doc('#doc-pending-fallback')))
```

Comment:
Here should be `eq_(1, len(doc('#doc-pending-fallback')))` because The fallback message will show! ;)
As the revision is ready for localization